src/components/Species/index.tsx

```diff
@@ -276,6 +292,62 @@ export default function SpeciesList({ reloadData, species }: SpeciesListProps):
     ];
   }, [activeLocale]);
 
+  const filterColumns = useMemo<FilterField[]>(
+    () => [
+      { name: 'growthForm', label: strings.GROWTH_FORM, type: 'multiple_selection' },
+      { name: 'conservationStatus', label: strings.CONSERVATION_STATUS, type: 'multiple_selection' },
+      { name: 'seedStorageBehavior', label: strings.SEED_STORAGE_BEHAVIOR, type: 'multiple_selection' },
+      { name: 'ecosystemTypes.ecosystemType', label: strings.ECOSYSTEM_TYPE, type: 'multiple_selection' },
+    ],
+    []
+  );
+
+  const [filters, setFilters] = useState<Record<string, any>>({});
+  const [filterOptions, setFilterOptions] = useState<FieldOptionsMap>({});
+
+  useEffect(() => {
+    const getApiSearchResults = async () => {
+      const searchParams: SearchRequestPayload = {
+        prefix: 'species',
+        fields: ['id', 'growthForm', 'seedStorageBehavior', 'ecosystemTypes.ecosystemType'],
+        search: {
+          operation: 'and',
+          children: [
+            {
+              operation: 'field',
+              field: 'organization_id',
+              type: 'Exact',
+              values: [selectedOrganization.id],
+            },
+          ],
+        },
+        sortOrder: [{ field: 'id', direction: 'Ascending' }],
+        count: 1000,
+      };
+
+      const data = await SearchService.search(searchParams);
```

Comment:
It's likely I missed this before.. there's an endpoint that retrieves values for dropdowns. See how we do this for accessions table https://github.com/terraware/terraware-web/blob/main/src/services/SeedBankService.ts#L142 may need some refactor to move it out into search.

Reply:
I'll put that as a followup! Thanks